adapters/IronSource/IronSourceAdapter/GADMAdapterIronSourceRtbRewardedAd.h

```diff
@@ -0,0 +1,25 @@
+//
```

Comment:
Please update. Same comment as at the top of GADMAdapterIronSourceRtbInterstitialAd.h. Thanks!

Reply:
Done